src/OrbitGl/TimeGraph.cpp

```diff
@@ -986,6 +991,15 @@ bool TimeGraph::IsVisible(VisibilityType vis_type, uint64_t min, uint64_t max) c
   }
 }
 
+void TimeGraph::SetVerticalScrollingOffset(float value) {
+  float clamped_value =
+      std::max(std::min(value, GetHeight() - viewport_->GetVisibleWorldHeight()), 0.f);
```

Comment:
Nit: std::clamp

Reply:
I remember that there was a reason why I chose max(min()) over clamp, but I can't recall... the result is not the same in all cases IIRC.

Reply:
Yes, the difference is when min > max in std::clamp (value, min, max). In that case, your code is returning max and clamp returns min.

Reply:
Yes, the difference is when lo > hi in std::clamp (value, lo, hi). In that case, your code is lo and std::clamp in undefined behavior. So, the issue here is if GetHeight() could be smaller than the VisibleWorldHeight(), am I correct?

Reply:
Ah, yes, that was the reason, thanks. It may happen that `GetHeight()` is smaller than `GetVisibleWorldHeight()`, but it may also be the other way around.